Makefile

```diff
@@ -0,0 +1,48 @@
+VERSION = v2.3.1.160220
```

Comment:
Hah, I wanted to do that today. Thanks! :smiley: 

Is that compatible with all different make binaries out there though?